src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -221,22 +216,16 @@ public void drive(double x, double y, double rotation) {
         }
 
         public void setRotationTarget(Rotation2d rotationTarget) {
-                if(!forceRotationTarget) {
-                        this.rotationTarget = rotationTarget;
-                }
-        }
-
-        public void setForceRotationTarget(boolean forceRotationTarget) {
-                this.forceRotationTarget = forceRotationTarget;
+                this.rotationTarget = rotationTarget;
         }
 
-        public boolean isRotationTargetForced() {
-                return forceRotationTarget;
+        public Rotation2d getRotationTarget() {
+                return rotationTarget;
         }
 
         @Override
         public void periodic() {
-                if(rotationTarget != null) {
+                if(chassisSpeeds.omegaRadiansPerSecond == 0) {
```

Comment:
This could be a problem at certain points during autonomous (if rotation is 0) therefore would like to still have the concept of rotationTarget = null as being off. The rotatationTarget should have null as its initial value and to be null when its disabled.

Additionally I think we want to reference point to be robot pose instead of the gyro rotation